templates/sync-template.yaml

```diff
@@ -16,20 +16,20 @@ spec:
           - name: "THOTH_DOCUMENT_ID"
           - name: "THOTH_FORCE_SYNC"
         artifacts:
-          - name: "document"
-            path: "/mnt/component/document/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: outputdocument
+            path: "/mnt/output/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
```

Comment:
so this is an **input** artifact, which is denoted as **output** document and is mounted in `/mnt/output`, that is quite confusing :D

Reply:
I thought from the point of view of a component which build an output, taken as input by graph-sync :)

Reply:
Ye, gotcha :) it's just a nitpick. I thought we might wanna follow a semantics of some sort and mount inputs to `/mnt/inputs` and outputs to `mnt/outputs`, WDYT?